src/icons/PlusIcon.tsx

```diff
@@ -0,0 +1,44 @@
+import type { h } from "preact";
+
+export default function PlusIcon(
```

Comment:
To the future: find a good icon library like that one TailwindUI uses:

`import { BoltIcon, ChatBubbleBottomCenterTextIcon, GlobeAltIcon, ScaleIcon } from '@heroicons/react/24/outline`

Reply:
Totally agree with.